manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/pages/AuthUserAddGroupPage.groovy

```diff
@@ -17,8 +18,8 @@ public class AuthUserAddGroupPage extends Page {
     }
 
     void choose(String group) {
-        def option = selectOption { $('select') }
-        selectOption.find("option").find { it.text() == group }.click()
+        selectOption = group
+        selectOption << Keys.ENTER
```

Comment:
we now manipulate the `input` field instead - need to press enter to select the option though